### ISSUE TITLE
Ability to download multiple videos in parallel?

### ISSUE THREAD

**Issue by nekohayo** [Unrud/video-downloader#74]
I often send a bunch of tabs/links of videos I want to archive on a particular computer when I come back to the office, and I like this app, for various reasons, over doing it on the commandline. However, in that scenario, I often want to download a dozen or so videos, and I'd like to not have to sit in front of the app and wait for each video to finish downloading before I can grab the next tab and download the next one, etc.

Some sort of batch / queuing feature would be very nice to have!

Some random ideas: when you start a download you also have a button that says "Queue another?"; or maybe simply having the UI duplicated somehow (i.e. the progress not replacing the existing gui, but being shown in an infobar/progressbar above/below the interactive GUI)

**Comment by Unrud** [comment]
You can just open multiple windows.

**Comment by ovari** [comment]
![Tube Converter](https://user-images.githubusercontent.com/17465872/196122775-14bb3d60-03fa-41e9-ace9-1b74ff987de3.png)

https://github.com/nlogozzo/NickvisionTubeConverter

**Comment by hyuri** [comment]
This would be excellent.

**Comment by maymage** [comment]
@Unrud 

> You can just open multiple windows.

An `+` button in the header bar to open a new window would be helpful then

**Comment by Michael-Lundholm** [comment]
Assuming you have a bash terminal and a file with links each on their own line in the current directory, you can run the command below then click download on all of the windows that pop up.

```bash
while read link ; do (video-downloader -u "$link" &); done < ./vids.txt
```
Replace `./vids.txt` with the location of your file with links on separate lines.
